WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsViewController.swift

```diff
@@ -1,12 +1,13 @@
 import UIKit
 import Combine
+import Yosemite
 
 final class CardReaderSettingsViewController: UIViewController {
 
     @IBOutlet weak var tableView: UITableView!
 
     private var subscriptions = Set<AnyCancellable>()
-    private var viewmodel = CardReaderSettingsViewModel()
+    private var viewModel = CardReaderSettingsViewModel()
```

Comment:
We'll inject this VM in a subsequent PR

Reply:
same comment here about declaring it as let instead of var